Src/BootCamp.Chapter/Equipment.cs

```diff
@@ -0,0 +1,91 @@
+﻿using BootCamp.Chapter.Items;
+
+namespace BootCamp.Chapter
+{
+    /// <summary>
+    /// Extra assignment. 
+    /// </summary>
+    public class Equipment
+    {
+        private Weapon _weapon;
+        public void SetWeapon(Weapon weapon)
+        {
+
+        }
+
+        private Headpiece _head;
+        public void SetHead(Headpiece head)
+        {
+
+        }
+
+        private Chestpiece _chest;
+        public void SetChest(Chestpiece chestpiece)
+        {
+
+        }
+
+        private Shoulderpiece _leftShoulder;
+        public void SetLeftShoulder(Shoulderpiece should)
+        {
+
+        }
+
+        private Shoulderpiece _rightShoulder;
+        public void SetRightShoulder(Shoulderpiece shoulder)
```

Comment:
Codacy found an issue: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5776099)